public/framework/catalog_cache/cache_loader.tsx

```diff
@@ -189,6 +188,7 @@ export const useLoadToCache = (loadCacheType: LoadCacheType) => {
   }, ASYNC_POLLING_INTERVAL);
 
   const startLoading = (dataSourceName: string, databaseName?: string) => {
+    setLoadStatus(DirectQueryLoadingStatus.SCHEDULED);
```

Comment:
is this meaning that, `SCHEDULED` means loading? For example, instead of checking if it is `SUCCESS`, `FAILED`, or `CANCELLED`, I can use this as an indicator to see if it is at a loading/refreshing stage?

Reply:
this is added as we want to start with status as scheduled whenever the `startpolling` is called. Otherwise, it keep the old calls status.

Reply:
`SCHEDULED` is the initial state, it may not necessarily be loading. 
```
const { loadStatus, startLoading, stopLoading } = useLoadAccelerationsToCache();
```
If `startLoading` isn't called, then `loadStatus` will remain as `SCHEDULED`.